staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -174,5 +183,9 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"handler, which picks a randomized value above this number as the connection timeout, "+
 		"to spread out load.")
 
+	fs.BoolVar(&s.EnableInfightQuotaHandler, "enable-inflight-quota-handler", s.EnableInfightQuotaHandler, ""+
+		"If true, replace the inflight limiting handler with the quota-based inflight handler which manages requests"+
```

Comment:
added a new flag here. any thought on the flag description?

Reply:
I think the word "quota" is actually a bit of a left turn here; we used other words more commonly in our discussions.  The word "quota" is not usually used in a context with "borrowing".  It is true that our initial take on this will have no borrowing, but that may be just an initial thing.  When the word "quota" is used, it is usually stated in absolute terms; in our initial proposal, what appears in the configuration is not absolute quantities but rather "shares".

Regardless of the name, I think a better description would be something like "replace the max-in-flight request handler with one that queues and dispatches with priority and fairness".

Reply:
"quota" is probably my fault; it's the generalized term used for this concept in all internal-google systems.

I think this is good enough for now, we can always change it later :)

Reply:
updated the description a bit wrt mike's suggestion